pedantic code warning: xbright.c:93:5: warning: ignoring return value of ‘fscanf’ declared with attribute ‘warn_unused_result’

allan@L4:/opt/xbright$ make && sudo make install
    cc -Wall -std=c99 -pedantic -g -DDEBUG -o obj/xbright.o -c xbright.c
    cc -g -o xbright obj/xbright.o
    [sudo] password for allan: 
    rm -f *.core
    rm -f build_host.h
    rm -f xbright
    rm -rf ./obj
    cc -Wall -std=c99 -pedantic -Os -o obj/xbright.o -c xbright.c
    xbright.c: In function ‘get_current’:
    xbright.c:93:5: warning: ignoring return value of ‘fscanf’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
       93 |     fscanf(input, "%d", &value);
          |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~
    cc  -o xbright obj/xbright.o
    strip xbright
    install -o root -g root -m 4755   xbright /usr/local/bin/
    DONE

--- a/xbright.c
+++ b/xbright.c
@@ -90,7 +90,7 @@ get_current(void)
     if (input == NULL) {
         ERROR("Cannot open kernel pipe");
     }
-    fscanf(input, "%d", &value);
+    (void)fscanf(input, "%d", &value);
     DBG("current value: %d", value);
     return value;
 }

